### PR TITLE
Adds "Segmented Visitor Log" icon to more reports and subtables

### LINKED ISSUE #7213: Display the "Segmented Visitor Log" icon in sub-tables reports
The goal of this issue is to display the "Segmented Visitor Log" icon in the sub-tables reports.

For example the following sub-tables should display the icon:
 * Search engines > Keyword
 * Keyword >  Search engine
 * Websites > Websites URL 
 * Campaign name -> Campaign keywords
 * (and other sub-tables possibly not displaying the icon yet?)

This follows up #4633 and is related to #4633

### LINKED ISSUE #9501: New segment DeviceBrand
Let users segment by device brand. (will be fixed with #9478)

### PR DESCRIPTION
**Pull request by sgiehl**
- Adds "Segmented Visitor Log" icon to various reports and subtables  (fixes #7213)
  - keywords > search engine
  - search engines > keyword
  - websites > wensite url
  - social networks > social network url
  - referrer type
- Adds new segment for devices brand  (refs #7212)

fixes #9501

### CODE REVIEW

File: tests/PHPUnit/System/expected/test_twoVisitsWithCustomVariables_segmentMatchNONE__CustomVariables.getCustomVariables_day.xml [hunk 1]
```diff
@@ -1,7 +1,279 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <results>
 	<result idSite="1">
-		<result date="2010-01-03" />
+		<result date="2010-01-03">
```
Comment (tsteur) [inline tests/PHPUnit/System/expected/test_twoVisitsWithCustomVariables_segmentMatchNONE__CustomVariables.getCustomVariables_day.xml]: I think you might need to modify https://github.com/piwik/piwik/blob/2.16.0-b1/tests/PHPUnit/System/TwoVisitsWithCustomVariablesSegmentMatchNONETest.php#L58-L77 and maybe add a special case so it doesn't match anything?

Comment (sgiehl) [inline tests/PHPUnit/System/expected/test_twoVisitsWithCustomVariables_segmentMatchNONE__CustomVariables.getCustomVariables_day.xml]: Good point. I'll try to adjust that.

**Comment by tsteur** [comment]
PR looks good! Can you maybe squash the commits and then merge? If not a direct merge should be ok too

File: tests/PHPUnit/System/expected/test_AutoSuggestAPITest_deviceBrand__API.getSuggestedValuesForSegment.xml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<result>
+	<row>Unknown</row>
```
Comment (mattab) [inline tests/PHPUnit/System/expected/test_AutoSuggestAPITest_deviceBrand__API.getSuggestedValuesForSegment.xml]: @sgiehl would it be possible to have this test generate real values so it shows that the auto suggest for this new segment works as expected?

Comment (sgiehl) [inline tests/PHPUnit/System/expected/test_AutoSuggestAPITest_deviceBrand__API.getSuggestedValuesForSegment.xml]: I'll issue a pull request for that...

**Pull request merged**